### PR TITLE
benjamin shen/dining

### PR DESCRIPTION
**Pull request by benjamin-shen**
### Summary 

This pull request is the first step towards implementing menus on Flux.  

- [x] implemented `getAllMenus()` => object containing relevant parts of the Cornell Dining API
- [x] implemented `getEateryMenus(slug)` => array containing all the menus of one eatery
- [x] implemented `getEateryUpcomingMenu(slug,time)` => object with information about one eatery's upcoming menu
- [x] implemented `getEateryDayMenus(slug,time)` => object with information about one eatery's menus throughout the day
- [x] implemented `getAllMealItems(slug,time)` => flattened array containing all the items served in one eatery's upcoming menu
- [x] implemented `getAllItems()` => set containing all the items served on menus this week

### Test Plan 

1. Uncomment `// test();` on line 198
2. Run `node functions/dining` on command line
3. Change slugs and observe console messages

### Notes 

- `dining.js` is currently a standalone file and is not integrated with the rest of Flux or flux-functions

### CODE REVIEW

**Review by SamChou19815 — DISMISSED** [review]
@benjamin-shen You should fix the linter errors first.
You can either see them from the `Files changed` tab in the PR page, or run `yarn lint` locally.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4627 @@
+{
```
Comment (SamChou19815) [inline package-lock.json]: This project currently uses Yarn. You should also use yarn to be consistent and remove the pacakge-lock.json.
@ashneeldas2 any opinions.

Comment (benjamin-shen) [reply package-lock.json]: Agreed, my mistake.

**Comment by ashneeldas2** [comment]
These changes will be quite valuable moving forward! We'll aim to use all of the work you've done in setting up the endpoint for the frontend this weekend.

**Review by ashneeldas2 — APPROVED** [review]
These have now been resolved – CI passes and linting errors no longer exist

**Pull request merged**